### DISCUSSION TITLE
Not able to create Link elements to web pages in Hubs Cloud

### DISCUSSION THREAD

**Discussion by leonardoazzi** [mozilla/hubs#5612] [Hubs Cloud Questions] [answered]
**Description**
In a Hubs Cloud room, links pointing to web pages appear as broken and are not accessible in the room.

**To Reproduce**
1. Enter a room in a Hubs Cloud instance;
2. Paste a page URL in the scene or go to 'Place' > 'Upload', paste the URL and click on 'Create Object';
3. See a Broken Link

**Expected behavior**
The page is fetched and the thumbnail is shown, along with an 'Open link' hover button that opens a new tab in the user's browser to that URL.

**Screenshots**

In my Hubs Cloud (db-vroom.com):

![Screenshot from 2022-07-25 18-22-24](https://user-images.githubusercontent.com/13557537/180878250-286b3d21-f6d7-498d-9948-3d7c46cedfd3.png)
![Screenshot from 2022-07-25 18-22-41](https://user-images.githubusercontent.com/13557537/180878235-7e1f9edf-cfc7-4e99-bee9-490755a6e8e0.png)
![Screenshot from 2022-07-25 18-22-48](https://user-images.githubusercontent.com/13557537/180878260-f4588d25-23a1-44ed-a241-a58a6fcffb91.png)

In Hubs (hubs.mozilla.com):
![Screenshot from 2022-07-25 13-45-43](https://user-images.githubusercontent.com/13557537/180877829-bcd346a0-8121-4121-a292-3079afbd21a8.png)

**Hardware** 
 - Device: Desktop
 - OS: Linux (Pop!_OS 22.04 LTS)
 - Browser: Chromium, Firefox
 
**Additional context**
It is important to state that have not merged the July hubs-cloud release to my custom code yet.

**Comment by pattersonbl2** [comment, marked as answer]
I think this could be related to the cloud formation not being updated.  When was the last time you updated it?

**Reply by leonardoazzi** [reply]
> I think this could be related to the cloud formation not being updated. When was the last time you updated it?

We were missing the latest cloudformation template. Now it works.

Thanks @Pattersonbl2 !
